tests/Fixtures/Integration/priority/fully_qualified_strict_types,no_superfluous_phpdoc_tags.test

```diff
@@ -8,14 +8,13 @@ Integration of fixers: fully_qualified_strict_types,no_superfluous_phpdoc_tags.
 /**
  * @param bool $expected
  */
-function testSomething(\Foo $foo, \Bar $bar, $expected) {}
+function testSomething(Foo $foo, $expected) {}
 
 --INPUT--
 <?php
 // No namespace
 /**
  * @param Foo  $foo
- * @param \Bar $bar
  * @param bool $expected
  */
-function testSomething(\Foo $foo, Bar $bar, $expected) {}
```

Comment:
keep deleted l18 and l21

Reply:
What value it brings? It was failing after reverting previous default behavior, so it must be modified anyway.